SUMMARY.md

```diff
@@ -152,7 +152,9 @@
 - [Content Search Indexer](libs/content-search-indexer/README.md)
 - [Content Search Metrics](libs/content-search-metrics/README.md)
 - [Content Search Toolkit](libs/content-search-toolkit/README.md)
-- [Contentful Translation Extension](libs/contentful-translation-extension/README.md)
+  - [Contentful Extension for Miðeind Machine Translations](libs/contentful-extensions/mideind-translation/README.md)
```

Comment:
You need a README at the root of the contentful-extensions folder to get the correct gitbook hierarchy https://docs.devland.is/repository/gitbook-contribute#root-readme-md-within-handbook-directory

Reply:
Thank you, I missed that :+1: